exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -347,6 +347,62 @@ func (ku *Kucoin) UpdateTradablePairs(ctx context.Context, forceUpdate bool) err
 	return nil
 }
 
+// GetFuturesTickers does n * REST requests based on enabled pairs of the futures asset type
+func (ku *Kucoin) GetFuturesTickers(ctx context.Context) ([]*ticker.Price, error) {
```

Comment:
Suggestion: shift to kucoin_futures.go as is not technically mapped to `exchange.IBotInterfaceface`